modules/lambdas/main.tf

```diff
@@ -72,8 +72,36 @@ module "restore_object_arn" {
 # Ingest Lambdas Definitions and Resources
 # =============================================================================
 
-# copy_to_glacier - Copies files to the ORCA S3 Glacier bucket
+# copy_to_glacier_cumulus_translator - Transforms input from Cumulus Dashboard to copy_to_glacier format
 # ==============================================================================
+resource "aws_lambda_function" "copy_to_glacier_cumulus_translator" {
+  ## REQUIRED
+  function_name = "${var.prefix}_copy_to_glacier_cumulus_translator"
+  role          = module.restore_object_arn.restore_object_role_arn
+
+  ## OPTIONAL
+  description      = "ORCA archiving lambda used to copy data to an ORCA S3 glacier bucket."
```

Comment:
Update the description for the Lambda please.

Reply:
Whoops. Fixed.